tests/test_raster.py

```diff
@@ -681,19 +757,25 @@ def test_downsampling(self, example: str) -> None:
 
             # - Test that xy2ij are consistent with new image -
             # Upper left
-            assert rst_down.xy2ij(rst_down.bounds.left, rst_down.bounds.top) == (0, 0)
+            assert rst_down.xy2ij(rst_down.bounds.left, rst_down.bounds.top, shift_area_or_point=False) == (0, 0)
```

Comment:
Just out of curiosity, here and all following tests, the argument `shift_area_or_point` is set to False. Is it only for this particular test that it is needed, or would it generally be needed? It would be strange to have to think about this when using xy2ij and hence the default argument should be the intuitive one.

Reply:
It's forced only for this particular test and a couple others, because we made assumptions based on "Area" (upper left) for the values of coordinates checked during the test.
Otherwise we shouldn't have to think about it, so the default is `True`. Most "Point" interpretation are for DEMs anyway, and we co-register them so that usually solves any difference :wink: